views/onepage.erb

```diff
@@ -11,23 +11,28 @@
 
   <% if @inline %>
 
-    <%= inline_css(['showoff.css', 'font-awesome-4.4.0/css/font-awesome.min.css', 'onepage.css', "highlight/#{@highlightStyle}.css"], 'public/css') %>
+    <%= inline_css(['font-awesome-4.4.0/css/font-awesome.min.css', 'mermaid-6.0.0.css', "highlight/#{@highlightStyle}.css"], 'public/css') %>
+    <%= inline_css(['showoff.css', 'onepage.css'], 'public/css') %>
```

Comment:
This looks like the same function called on two separate arrays.  Why isn't it one array and one call?